tasks/time-series/time-series-forecasting/main.py

```diff
@@ -0,0 +1,118 @@
+# Imports
+import numpy as np
+import tensorflow as tf
+from sklearn.preprocessing import MinMaxScaler
+import keras
+from keras.models import Model
+from keras.layers import Dense, CuDNNLSTM, Input
+import datetime
+import os
+import shutil
+from numpy.random import seed
+from tensorflow import set_random_seed
+import matplotlib.pyplot as plt
+import argparse
+import mlflow.sklearn
+from mlflow import log_metric
+import mlflow.keras
+import warnings
+import sys
+
+# Initialize
+# Keras and Tensorflow Random Numbers
+seed(1)
+set_random_seed(2)
+
+# Clear
+# Previous Tensorflow Session
+tf.keras.backend.clear_session()
+
+# Parse Arguments
+parser = argparse.ArgumentParser()
+parser.add_argument("city_name", help="Name of the city to get climate data", type=str)
+args = parser.parse_args()
+
+# Define Variables
+lookback_window = 32
+steps_per_epoch = 364
+val_steps = 364 * 30
+LSTMunits = 100
+predict_steps = 364 * 10
+predict_shift = 0
+city = sys.argv[1]
+
+# Working directories
+model_filepath = 'Weather_Net_'+city+'.hf5'
```

Comment:
Optional: Please create model as training step. waiting for 10 minutes should be ok. We are usually downloading pre trained models from original source which requires lot of resource but Weather_Net model is an outcome of this task and relatively cheaper to create. Please also update README.md accordingly.

Reply:
Done. Removed train_weathernet.py, keeping only main.py.

Reply:
I apologizes if you miss understood my comment. I wanted to say do not checking .hf5 model file but it's good to have 2 steps, one for model generation and second for prediction.

We can add additional check in prediction which can check if model exist or not and train model first in case model does not exist.

These can be added through multi step workflow.

Reply:
No problem, commited both files back. One for training only (train_weathernet) and one for prediction (main). In case pretrained model is not available, training is done. Currently only as a simple check since I need to dig deeper into MLflow multistep workflow.